central/processindicator/datastore/datastore_impl.go

```diff
@@ -153,26 +153,46 @@ func (ds *datastoreImpl) removeIndicators(ctx context.Context, ids []string) err
 		return err
 	}
 
+	if err := ds.indexer.DeleteProcessIndicators(ids); err != nil {
```

Comment:
I am assuming you changed the order since postgres is turned on by default and this op is essentially a noop.

Reply:
exactly.